### PR TITLE
Stability hotfix

### LINKED ISSUE #1024: Crashes
**Your mod version is:**
the 1.5 master branch
**What expansions do you have installed?**
Garments of the holy roman empire, and northern lords
**Are you using any submods/mods? If so, which?**
only GoA
**Please explain your issue in as much detail as possible:**
After playing as any character or in observer mode the game will crash after a certain amount of time.
**Steps to reproduce the issue:**
Just let the game run, it usually happens at some point after the year 650ish. 
**Upload an attachment below: .zip of your save, or screenshots:**
[ck3_20230307_154852.zip](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/files/10913811/ck3_20230307_154852.zip)

### PR DESCRIPTION
**Pull request by VespertinePetrichor**
Closes #1024 
## Changelog:
- Updated Mod Version to Release Alpha
- Fixed short dragon lifespans
- Fixed title history errors
- Age scaling of peasant revolt leaders temporarily disabled
- Resolves trigger, scope, unused tags and syntax errors
- Adapted events:
  - Foreign prince `court.0108` 
  - Syncretic faith conversion `hold_court.8120`
  - `Arabic Book Template`
- Removed unused vanilla kabarism event and modifiers. 
- Added Vanilla religions to mod files, because mod was not reading vanilla and that was causing errors on a lot of files that still use Pagan, Eastern, etc.

### Instructions:
Open a new game and let it run on five speed for a few years. Check error log for looping errors that will destablise a save game and bloat the log, causing slowdowns and CTDs. Resolve any other errors at your discretion.

## Developer changelog:
- [x]   Error: Event target link 'capital_province' did not get a matching scope type. Expected 'character', but got 'culture'
  Script location: file: common/decisions/10_culture_conversion_decisions.txt line: 231

### 3rd Bookmark (Looping Errors)
- [x]   Error: Invalid right side during comparison 'cp'
  Script location: file: gfx/portraits/portrait_modifiers/01_headgear.txt line: 967
  - **How to replicate:** Select Ner'zhul and browse to his courtiers. Run the game for a few moments.
- [x]  Error: calc_culture_dominant_religion trigger [ Failed context switch ]
  Script location: file: common/script_values/01_starting_values.txt line: 169 
   - Not sure what is causing this error, as all associated code is unaltered from its vanilla form.

### 1st Bookmark (Looping Errors)
- [x]   Error: scope:culture trigger [ Failed context switch ]
  Script location: file: common/scripted_effects; random_dummy_gender_landholder_effect line: 10 file: events/lifestyles/governance_lifestyle/stewardship_wealth_events.txt line: 851
- [x]   Error: has_trait trigger [ Wrong scope for trigger: landed title, expected character ]
  Script location: file: common/scripted_triggers; has_race_trait_trigger line: 2 file: events/factions/faction_demands.txt line: 566 file: common/scripted_effects; successful_popular_revolt_outcome_effect line: 313 file: common/script_values/wc_age_values.txt line: 43 file: common/scripted_triggers; is_race_no_gene_trigger line: 3
- [x] **Often precedes CTDs** Trying to add the temporary target 'temp_culture' which has the same name as a permanent target, at  file: common/scripted_triggers; like_polish_culture_trigger line: 7
---
~~**Often precedes CTDs** [titleandvassalchangemanager.cpp:24]: TitleAndVassalChange that hasn't been resolved originating from Unknown Source~~

~~Error: Event target link 'scope' returned an unset scope
Script location: file: common/scripted_triggers; artifact_low_rarity_trigger line: 2 file: common/artifacts/features/00_features.txt line: 6081 file: common/scripted_triggers; artifact_value_trigger line: 13~~

## Tests:
- [x] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [x] The mod takes less than 5.5 GB in the Task Manager (Windows)

# How to test:
1) Let the game run to at least 950 as either an observer. Run from all 3 bookmarks please.
2) Have some fun! We need people to do a playthrough too, to test specific interactions and mechanics that are not triggered as an observer.

### CODE REVIEW

**Review by TheGroxEmpire — CHANGES_REQUESTED** [review]
First bookmark I experienced a crash. I was running at 5 speed and zooming the map out and in.
[error.log](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/files/11007801/error.log)
[first_bookmark_crash.zip](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/files/11007822/first_bookmark_crash.zip)

**Comment by TheGroxEmpire** [comment]
I did tests on all three bookmark again after the recent commits.
First bookmark crashed just as quickly as last time.
[2_first_bookmark_crash.zip](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/files/11010569/2_first_bookmark_crash.zip)

Second bookmark is the most stable, I was able to run until year 702.
[2_second_bookmark_crash.zip](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/files/11010570/2_second_bookmark_crash.zip)

Third bookmark is a bit more stable with crash at year 628.
[2_third_bookmark_crash.zip](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/files/11010571/2_third_bookmark_crash.zip)

File: events/court_events/court_events_general.txt [hunk 1]
```diff
@@ -542,15 +542,17 @@ court.0108 = {
...
-				OR = {
-					like_islam_religion_trigger = yes
-					has_doctrine_parameter = islamic_syncretic_actor_opinion_active
-				}
+			target = scope:foreign_prince.faith = {
+				has_faith = faith:cult_of_wealth
+				# OR = {
+					# like_islam_religion_trigger = yes
+					# has_doctrine_parameter = islamic_syncretic_actor_opinion_active
+				# }
```
Comment (zumbak04) [inline events/court_events/court_events_general.txt:555]: Syntax errors? Never seen such syntax in vanilla.

If it's supposed to check if `scope:foreign_prince.faith` is `faith:cult_of_wealth`, I believe should look like this: `scope:foreign_prince.faith = faith:cult_of_wealth`.

File: events/lifestyles/governance_lifestyle/stewardship_wealth_events.txt [hunk 2]
```diff
@@ -1880,7 +1880,8 @@ stewardship_wealth.6001 = {
 		is_at_war = no
 		any_held_county = { count >= 1 }
 		NOT = {
-			faith = faith:iconoclast
+			#Warcraft
+			#faith = faith:iconoclast
 		}
```
Comment (zumbak04) [inline events/lifestyles/governance_lifestyle/stewardship_wealth_events.txt:1885]: Comment the entire NOT block. There's no need for NOT block without conditions.

**Review by zumbak04 — APPROVED** [review]
I left comments, once they are fixed I'm good with fixes.

**Comment by zumbak04** [comment]
> I did tests on all three bookmark again after the recent commits. First bookmark crashed just as quickly as last time. [2_first_bookmark_crash.zip](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/files/11010569/2_first_bookmark_crash.zip)
> 
> Second bookmark is the most stable, I was able to run until year 702. [2_second_bookmark_crash.zip](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/files/11010570/2_second_bookmark_crash.zip)
> 
> Third bookmark is a bit more stable with crash at year 628. [2_third_bookmark_crash.zip](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/files/11010571/2_third_bookmark_crash.zip)

Don't forget to disable other mods, I see some errors coming from this.

File: common/decisions/80_major_decisions.txt [hunk 3]
```diff
@@ -19,7 +19,7 @@
 #	desc = convert_to_feudalism_decision_desc
 #	
 #	ai_check_interval = 36
-#	
+#	witen
```
[the remark this answers is not in the document]
Comment (VespertinePetrichor) [reply common/decisions/80_major_decisions.txt:22]: Removed it

File: events/court_events/court_events_general.txt (hunk 1 above)
Comment (VespertinePetrichor) [reply events/court_events/court_events_general.txt:555]: I attempted this solution but it gives errors. Try searching `target = scope:` in \game\events` in the vanilla ck3 directory. There are quite a lot time where `target = scope:` points to a temporary scope defined first in that file. 

e.g. in `death_management_events.txt` there are these lines:
> [Line 8:] 	this = scope:dead_character
> [Line 1425] trigger = {	scope:dead_character.highest_held_title_tier >= root.highest_held_title_tier }

**Comment by TheGroxEmpire** [comment]
> > I did tests on all three bookmark again after the recent commits. First bookmark crashed just as quickly as last time. [2_first_bookmark_crash.zip](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/files/11010569/2_first_bookmark_crash.zip)
> > Second bookmark is the most stable, I was able to run until year 702. [2_second_bookmark_crash.zip](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/files/11010570/2_second_bookmark_crash.zip)
> > Third bookmark is a bit more stable with crash at year 628. [2_third_bookmark_crash.zip](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/files/11010571/2_third_bookmark_crash.zip)
> 
> Don't forget to disable other mods, I see some errors coming from this.

I double checked and I didn't have any other mods enabled.

File: events/court_events/court_events_general.txt (hunk 1 above)
Comment (zumbak04) [reply events/court_events/court_events_general.txt:555]: > I attempted this solution but it gives errors.

Yeah, that makes sense. The scope may be not set in the modifier block. In the vanilla event, there is no scoping used.
<details>
<summary>Click to expand</summary>

image

</details>

> > [Line 1596] target = scope:dead_character

That's a part of the opinion = {} trigger syntax.

<details>
<summary>Click to expand</summary>

image

</details>

Comment (VespertinePetrichor) [reply events/court_events/court_events_general.txt]: Addressed in 7cc70ea8e

**Pull request merged**